File: examples/php/image_upscale.php
<?php

declare(strict_types=1);

require __DIR__ . '/../../packages/php/vendor/autoload.php';

use AILabTools\AILabClient;
use AILabTools\Generated\Params\ImageLosslessEnlargementParams;
use AILabTools\Http\FileInput;

$client = new AILabClient(apiKey: getenv('AILAB_API_KEY') ?: '');
$result = $client->image->upscale(
    new ImageLosslessEnlargementParams(
        image: FileInput::fromPath(__DIR__ . '/../../image.jpg'),
        upscaleFactor: 2,
    ),
);

echo $result->data?->url . PHP_EOL;

File: examples/dotnet/ImageUpscale.cs
using AILabTools;
using AILabTools.Generated.Params;
using AILabTools.Http;

using var client = new AILabToolsClient(
    Environment.GetEnvironmentVariable("AILAB_API_KEY")!);

var result = await client.Image.UpscaleAsync(
    new ImageLosslessEnlargementParams
    {
        Image = FileInput.FromPath("./image.jpg"),
        UpscaleFactor = 4,
        OutputFormat = "jpg",
    });

Console.WriteLine(result.Data?.Url);
